added mux with out any sources (#16667)
tldr-pages/tldr: pages/linux/udevmon.md

# udevmon

> > An interception tool for monitoring input devices for launching tasks.

Reviewer: ```suggestion
> Intercept and monitor input devices for launching tasks.
```
